docs/pages/server-access/guides/teleport-ssh.mdx

```diff
@@ -0,0 +1,241 @@
+---
```

Comment:
Same here, no need for `teleport-ssh`.  The purpose of this section is not clear compared to getting started. Is that a guide on access requests or something else? I think we need to narrow the scope of this particular guide.